public/rulesets/default/2_3_2_Content_Type

```diff
@@ -0,0 +1,24 @@
+rules:  
+  - id: "2.3.2"
+    title: "Content-Type: Default Media Type"
+    message: "If the request or response body is present, the default media type must be application/json with charset=UTF-8 or application/bson."
+    option: Mandatory
+    location: path
+    element:
+  	- requestBody
+  	- responses
+    call:
+  	function: mediaTypeCheck
+  	functionParams:
+  	  headers:
+  		- "application/json; charset=UTF-8"
```

Comment:
change to Content-Type: application/json

           Content-Type:
              schema:
                type: string

https://swagger.io/docs/specification/v3_0/describing-responses/#response-headers